<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentform</class>
 <widget class="QWidget" name="studentform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>个人信息</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>密码修改</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>旧密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>新密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnQuit_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOk_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNewPwd2_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leOldPwd_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNewPwd_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>开始测试</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>测试题</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>题号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>180</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>所用时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>250</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>所得分数：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>正确答案：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>280</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>你的答案：</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>291</width>
         <height>42</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rBtnA_3">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="rBtnB_3">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="rBtnC_3">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="rBtnD_3">
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="leAns_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>280</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leYourAns_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>280</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leQuestionID_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>110</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leTime_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>180</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leScore_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>240</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrevious_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>上一题</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnNext_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>320</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>下一题</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStart_3">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>286</y>
         <width>97</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSubmit_3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>286</y>
         <width>97</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbxSub_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>30</y>
         <width>111</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C++</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Java</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C#</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
